resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -796,6 +811,38 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
       None
     }
   }
+
+  // Calls used for metrics polling, see MesosCoarseGrainedSchedulerSource:
+
+  def getCoresUsed(): Double = totalCoresAcquired
```

Comment:
Are there any issues with concurrency here. When

Reply:
In practice there shouldn't be. In the case of a race, values could be briefly out of date but they'd be fine on the following refresh. This structure mirrors the dispatcher in `MesosClusterScheduler.scala`, see e.g. `getQueuedDriversSize`, `getLaunchedDriversSize`, and `getPendingRetryDriversSize` in there.